sterling/tests/test_sterling.py

```diff
@@ -18,15 +18,16 @@
     "ipval": str(fake)
 }
 
+
 @patch.object(Session, 'request')
 def test_enquiry(mock_get):
     data = responses["enquiry"]
     mock_get.return_value = R(responses["enquiry"])
-    assert Enquiry(header).name_enquiry(query) == data, "should return an object"
+    assert Enquiry(header).InterbankNameEnquiry(query) == data, "should return an object"
```

Comment:
Change `assert Transfer(header).InterbankNameEnquiry(query) == data, "should return an object"`
To `assert Transfer(header).InterbankNameEnquiry(query) == data, "should return an object"`

Basically, rename Enquiry to Transfer here and anywhere else applicable.